stdlib/runtime/wasi.gr

```diff
@@ -161,3 +164,97 @@ let _ETIMEDOUT = 73n
 let _ETXTBSY = 74n
 let _EXDEV = 75n
 let _ENOTCAPABLE = 76n
+
+exception SystemError(Number)
+
+let stringOfSystemError = (code) => {
```

Comment:
This is honestly one the biggest things that I'm excited for.

Reply:
🔥